lighthouse-core/report/templates/report-template.html

```diff
@@ -187,9 +183,7 @@ <h2 class="config-section__title">Blocked URL Patterns</h2>
                   {{/if}}
 
                   <div class="subitem-result">
-                    {{#if subItem.comingSoon}}
-                          <span class="subitem-result__unknown score-unknown-bg">N/A</span>
-                    {{else if subItem.error}}
+                    {{#if subItem.error}}
```

Comment:
maybe remove `.subitem-result__unknown.` and `.score-unknown-bg` now that they're unused? are there other things we're missing?